### PR TITLE
3/4 Upstream Merge

### PR DESCRIPTION
**Pull request by Jolly-66**
base code should be 100% functional.

not sure how to get the modular jobs trims loaded, but the codes there
also need to add heirlooms to our custom jobs, again. thanks melbert.

### CODE REVIEW

**Comment by Jolly-66** [comment]
if everythings good base code wise, you have my permission to merge this melbert

**Comment by MrMelbert** [comment]
image
this shit bangin' yo.

File: jollystation_modules/code/game/machinery/cryopod.dm [hunk 1]
```diff
@@ -182,7 +182,7 @@
 	// These items are preserved when the process() despawn proc occurs.
 	var/static/list/preserve_items = typecacheof(list(
 		/obj/item/hand_tele,
-		/obj/item/card/id/captains_spare,
+//		/obj/item/card/id/captains_spare, -uhh, i don't know what it got replaced to
```
Comment (MrMelbert) [inline jollystation_modules/code/game/machinery/cryopod.dm:185]:
```suggestion
          /obj/item/card/id/advanced/gold/captains_spare
```

**Comment by MrMelbert** [comment]
does this actually compile? the 4/4 failing checks gives me doubts

**Comment by Jolly-66** [comment]
It compiles fine for me
Idk why the checks are here now after not having them for a month

**Comment by Jolly-66** [comment]
> image

image

🤔

**Comment by Jolly-66** [comment]
i think the checks are erroring because they're not running the jollystation code, but thats just me

i have 0 idea what these checks are supposed to do, and like i said, we didn't have them for a month until this upstream

**Comment by Jolly-66** [comment]
also ig it isn't showing up, but i did modify the og cards.dmi or w/e its called
modular file still exists, so if it does become an issue later (which it shouldn't due to this mega refactor) its as easy as just changing paths and we'll be fine

File: jollystation_modules/code/game/machinery/cryopod.dm [hunk 2]
```diff
@@ -182,7 +182,7 @@
 	// These items are preserved when the process() despawn proc occurs.
 	var/static/list/preserve_items = typecacheof(list(
 		/obj/item/hand_tele,
-		/obj/item/card/id/captains_spare,
+          /obj/item/card/id/advanced/gold/captains_spare,
```
Comment (MrMelbert) [inline jollystation_modules/code/game/machinery/cryopod.dm:185]: indentation got messed up here but whatever I guess

**Pull request merged**